src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorTclBinder.java

```diff
@@ -105,6 +105,8 @@ public void start() {
             // FIXME: hardcoded string
             final var errorMessage = new StringBuilder(
                     "You may disable VHDL simulation in the simulation menu if this occurs again\n\n");
+            
+            String msgCheckOnError = "\"" + VhdlSimConstants.VHDL_COMPILE_COMMAND + VhdlSimConstants.VHDL_COMPONENT_SIM_NAME;
```

Comment:
Please avoid "+" in string concatinations, Please use `String.format("\"%s%s",VhdlSimConstants.VHDL_COMPILE_COMMAND, VhdlSimConstants.VHDL_COMPONENT_SIM_NAME);` instead

Reply:
Done in the most recent commit